content/docs/instrumenting/writing_exporters.md

```diff
@@ -109,7 +85,7 @@ Prometheus metrics and label names are written in `snake_case`. Converting
 doesn’t always produce nice results for things like `myTCPExample` or `isNaN`
 so sometimes it’s best to leave them as-is.
 
-Exposed metrics should not contain colons, these are for users to use when
+Exposed metrics should not contain colons, these are reserved for use when
```

Comment:
It needs to be clear that these are for end-user use. "aggregating" has a few different meanings depending on who you're talking to.